README.md

```diff
@@ -6,7 +6,7 @@
 
 Node Module for performing retries for HTTP requests.
 
-It is a wrapper around [`node-fetch-npm`](https://github.com/npm/node-fetch-npm) library. It has default retry logic built in as described below, as well as configurable parameters. It also has built-in support for Apache OpenWhisk actions, adjusting the timeout to reflect the action timeout.
+It is a wrapper around [`node-fetch`](https://www.npmjs.com/package/node-fetch) library. It has default retry logic built in as described below, as well as configurable parameters. It also has built-in support for Apache OpenWhisk actions, adjusting the timeout to reflect the action timeout.
```

Comment:
Let's keep the link to the github repo rather than npm:

```suggestion
It is a wrapper around [`node-fetch`](https://github.com/node-fetch/node-fetch) library. It has default retry logic built in as described below, as well as configurable parameters. It also has built-in support for Apache OpenWhisk actions, adjusting the timeout to reflect the action timeout.
```